tests/core/actions/test_forms.py

```diff
@@ -1091,3 +1090,22 @@ async def test_ask_for_slot(
     )
 
     action_from_name.assert_called_once_with(expected_action, domain, endpoint_config)
+
+
+async def test_ask_for_slot_if_not_utter_ask(monkeypatch: MonkeyPatch):
+    action_from_name = Mock(return_value=action.ActionListen())
+    endpoint_config = Mock()
+    monkeypatch.setattr(
+        action, action.action_for_name_or_text.__name__, action_from_name
+    )
+
+    form = FormAction("my_form", endpoint_config)
+    await form._ask_for_slot(
+        Domain.empty(),
+        None,
```

Comment:
doesn't align with type annotations, this isn't an `Optional` parameter

Reply:
(I hope the type checker is complaining about it as well 😅)

Reply:
@wochinge could you please take this over